test/governance/candidacy.users.test.ts

```diff
@@ -0,0 +1,175 @@
+
+    await candidate.runForCouncil();
+    await waitForNBlocks(2);
+
+    const candidates = getLastBlocksElectionData();
+
+    expect(candidates).toEqual(expect.arrayContaining([candidatesAddress]));
+  });
+
+  test("Candidate can renounce candidacy", async () => {
+    const candidatesAddress = candidate.keyRingPair.address;
+
+    await candidate.runForCouncil();
+    await waitForNBlocks(2);
+    await candidate.renounceCandidacy();
+    await waitForNBlocks(2);
+
+    const candidates = getLastBlocksElectionData();
+
+    expect(candidates).not.toEqual(expect.arrayContaining([candidatesAddress]));
+  });
+
+  test("Council Member can renounce candidacy", async () => {
+    const candidatesAddress = candidate.keyRingPair.address;
+
+    // Gonzalo runs for Council in Term 0
+    // Eddy votes for Gonzalo
+    await candidate.runForCouncil();
+    await voter.vote([candidate], new BN(Math.pow(10, 16).toString()));
+    await waitForNBlocks(2);
+
+    const candidatesTerm0 = getLastBlocksElectionData();
+
+    expect(candidatesTerm0).toEqual(
+      expect.arrayContaining([candidatesAddress])
+    );
+
+    // Gonzalo wins the election and is council member!
+    // Gonzalo subsequently renounces candidacy in Term 1
+    await waitForNextTerm();
+    await candidate.renounceCandidacy();
+
+    // Gonzalo is not automatically running for candidacy in Term 2
+    await waitForNextTerm();
+
+    const candidatesTerm2 = getLastBlocksElectionData();
+
+    expect(candidatesTerm2).not.toEqual(
+      expect.arrayContaining([candidatesAddress])
+    );
+  });
+
+  test.only("Council Member automatically runs for candidacy again", async () => {
+    const candidatesAddress = candidate.keyRingPair.address;
+
+    // Gonzalo runs for Council in Term 0
+    // Eddy votes for Gonzalo
+    await candidate.runForCouncil();
+    await voter.vote([candidate], new BN(Math.pow(10, 16).toString()));
+    await waitForNBlocks(2);
+    const candidatesTerm0 = getLastBlocksElectionData();
+    expect(candidatesTerm0).toEqual(
+      expect.arrayContaining([candidatesAddress])
+    );
+
+    // Gonzalo wins the election and is a council member!
+    await waitForNextTerm();
+
+    const candidatesTerm1 = getLastBlocksElectionData();
+
+    expect(candidatesTerm1).toEqual(
+      expect.arrayContaining([candidatesAddress])
+    );
+
+    // Gonzalo is automatically running for candidacy in Term 2
+    await waitForNextTerm();
+
+    const candidatesTerm2 = getLastBlocksElectionData();
+
+    expect(candidatesTerm2).not.toEqual(
+      expect.arrayContaining([candidatesAddress])
+    );
+  });
+});
+
+async function waitForNextTerm() {
+  // Blocks per term are 50 on the modified mangata node
+  // We know the last block, so we can work out our position in the term
+  // as lastBlock % blocksPerTerm. We wait an additional 3 blocks purely to avoid
+  // tests needing to manually wait a few extra blocks too.
+  const blocksPerTerm = 50;
```

Comment:
why not reading the term duration from 
`( await api.derive.elections.info() ).termDuration` ?